### PR TITLE
Weborama Real-time Data Module: add new RTD module

### PR DESCRIPTION
**Pull request by peczenyj**
## Type of change

- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Weborama provides a Semantic AI Contextual API that classify in Real-time a web page seen by a web user within generic and custom topics. It enables publishers to better monetize their inventory and unlock it to programmatic advertising.

ORTB2 compliant and FPD support for Prebid versions < 4.29

- contact email of the adapter’s maintainer: prebid-support@weborama.com 
- [X] official adapter submission

### CODE REVIEW

File: modules/weboramaRtdProvider.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+| params | Object | | Optional |
+| params.weboCtxConf | Object | Weborama Contextual Configuration | Optional |
+| params.weboCtxConf.token | String | Security Token provided by Weborama, unique per client | Mandatory |
+| params.weboCtxConf.targetURL | String | Url to be profiled in the contextual api | Optional. Defaults to `document.URL` |
+| params.weboCtxConf.defaultProfile | Object | default value of the profile to be used when there are no response from contextual api (such as timeout)| Optional. Default is `{}` |
+| params.weboCtxConf.gamTargetingWeboCtxKey | String | allow rename the key `webo_ctx`  in gam targeting| Optional. Default is `webo_ctx`|
+| params.weboCtxConf.gamTargetingWeboDSKey | String | allow rename the key `webo_ds` in gam targeting | Optional. Default is `webo_ds`|
+| params.weboCtxConf.setTargeting|Boolean|If true, will use the contextual profile to set the gam targeting of all adunits managed by prebid.js| Optional. Default is *true*.|
+| params.weboCtxConf.setOrtb2|Boolean|If true, will use the contextual profile to set the ortb2 configuration on `site.ext.data`| Optional. Default is *true*.|
```
Comment (patmmccann) [inline modules/weboramaRtdProvider.md:62]: why not register for a segtax and set site.data ?

Comment (peczenyj) [reply modules/weboramaRtdProvider.md:62]: The API is still in development, we will probably do it in a near future

Comment (peczenyj) [reply modules/weboramaRtdProvider.md]: we decide to keep this feature not active by default. clientes will use at their own responsibility

File: modules/weboramaRtdProvider.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+ * @param {null|Object} data
+ * @returns {void}
+ */
+export function setBigseaContextualProfile(data) {
+  if (data && Object.keys(data).length > 0) {
+    _bigseaContextualProfile = data;
+  }
+}
+
+/** onSucess callback type
```
Comment (msm0504) [inline modules/weboramaRtdProvider.js:92]: Will `onSuccess` and `onDone` functions be added later? If not, can these comments be removed?

**Review by msm0504 — APPROVED** [review]

File: modules/weboramaRtdProvider.js (hunk 2 above)
Comment (peczenyj) [reply modules/weboramaRtdProvider.js]: I am using onSuccess and onDone on `fetchContextualProfile` but this function is not exported. I can remove if you want

**Comment by peczenyj** [comment]
@msm0504 @patmmccann do I need to add this module on `modules/.submodules.json` ?

**Comment by msm0504** [comment]
@peczenyj Yes, this should be added under `rtdModule` in `modules/.submodules.json`. This will ensure that the RTD Module will be included with it.

**Comment by peczenyj** [comment]
> @peczenyj Yes, this should be added under `rtdModule` in `modules/.submodules.json`. This will ensure that the RTD Module will be included with it.

Ok I add here
https://github.com/prebid/Prebid.js/pull/7461

But will be nice add this step explicit in the documentation. I think we have this only for id submodules and I did not check it

**Pull request merged**